### PR TITLE
ui improvement of sidebar 

### PR DESCRIPTION
**Pull request by ARJUN-SABU**
issue #3 :
Improved UI of side bar by:
1. Adding blue border to the playground title
2. Adding change of background-color of ListItem component when it is clicked showing that which component was clicked.
3. Changed inline css to css modules for footer and sidebar.

### CODE REVIEW

**Review by tewarig — COMMENTED** [review]
well done..
Screenshot from 2021-03-11 14-54-18
I was thinking it would be great if the playground and 🚀 comes under one line only. 
Screenshot from 2021-03-11 15-01-17

you did great work keep it up.

**Comment by ARJUN-SABU** [comment]
Oh thanks! But, when I am viewing the deployed version myself then the title and the rocket are on the same line and in your case its on different lines 🤔.

**Comment by pranshuchittora** [comment]
> well done..
> Screenshot from 2021-03-11 14-54-18
> I was thinking it would be great if the playground and  comes under one line only.
> Screenshot from 2021-03-11 15-01-17
> 
> you did great work keep it up.

It's coming in one line, I think are zoomed-in kindly reset the page zoom level to 100%

File: src/containers/Drawer/ResponsiveDrawer.module.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.brand_name {
+  font-weight: 100;
+  padding: 8px;
+  border-bottom: 3px solid #2089dc;
```
Comment (pranshuchittora) [inline src/containers/Drawer/ResponsiveDrawer.module.css:4]: We can replace this with a <Divider> from mui/core

**Review by pranshuchittora — CHANGES_REQUESTED** [review]

File: src/containers/Drawer/ResponsiveDrawer.module.css (hunk 1 above)
Comment (ARJUN-SABU) [reply src/containers/Drawer/ResponsiveDrawer.module.css:4]: Ok, I will add those styles from css module to makeStyles of material-ui/core/styles

**Comment by ARJUN-SABU** [comment]
1. Changed the css module styles of title to material-ui/core/styles format
2. Added blue coloured border-right when hovering over side elements and selecting any one of them, indicating which element was selected.

The side bar

File: src/containers/Drawer/index.jsx [hunk 2]
```diff
@@ -57,13 +57,31 @@ const useStyles = makeStyles((theme) => ({
     paddingTop: "1.5rem",
     minHeight: "100vh",
   },
+
+  brand_name: {
+    fontWeight: "100",
+    padding: "8px 0",
+    marginLeft: "5px",
+    borderBottom: "3px solid #2089dc",
```
Comment (pranshuchittora) [inline src/containers/Drawer/index.jsx:65]: Pls remove this & replace with a <Divider>

**Review by pranshuchittora — CHANGES_REQUESTED** [review]

File: src/containers/Drawer/index.jsx (hunk 2 above)
Comment (ARJUN-SABU) [reply src/containers/Drawer/index.jsx:65]: Ok removing it.

**Comment by ARJUN-SABU** [comment]
SideBar update_2

Removed the above mentioned code.
Is it ok?

**Comment by pranshuchittora** [comment]
Thanks :)

**Pull request merged**